### PR TITLE
Features/157610021/invite notification ui design

### PR DESCRIPTION
**Pull request by Orlayhemmy**
#### What does this PR do?
It re-designs the UI for sending invites to online users

#### Description of Task to be completed?
- add tabs to hold players in the game and online users

#### What are the relevant pivotal tracker stories?
Story Id #157610021

#### Screenshots (if appropriate)
screen

screen shot 2018-05-16 at 5 09 30 pm

### CODE REVIEW

File: public/views/app.html [hunk 1]
```diff
@@ -1,165 +1,214 @@
...
-        </div>
+  </div>
+  <!-- User Found Modal-->
+  <div id="modal" ng-show="game.message">
+    <div id="modal-content">
+      <button type="button" ng-click="closeModal()">&times;</button>
+      <div class="modal-body" ng-if="game.message">
+        <h6>{{game.message}}</h6>
+      </div>
+      <div class="button-div">
```
Comment (amarachukwu-agbo) [inline public/views/app.html:194]: The closing div tags seem to be more than the opening div tags. Did you addd an extra one?

**Review by amarachukwu-agbo — COMMENTED** [review]
You have done a good job implementing this feature however, I feel there are areas that can be improved on. Currently, the notification bell displays even when a user does not have a notification and it displays an empty box when clicked. I suggest you hide the div using angular directives like **ng-show** or **ng-if** if there are no notifications or display a message in the div like **You currently have no notifications**, that way, the user has a better experience.

File: public/views/app.html (hunk 1 above)
Comment (Orlayhemmy) [reply public/views/app.html:194]: Thanks, I will check that out and fix it

Comment (Orlayhemmy) [reply public/views/app.html:194]: Thanks will check that out

**Comment by Orlayhemmy** [comment]
I have implemented your feedback

**Pull request closed without merging**